templates/spago.dhall

```diff
@@ -1,6 +1,15 @@
 {-
 Welcome to a Spago project!
 You can edit this file as you like.
+
+Need help? See the following resources:
+- Spago project:        https://github.com/purescript/spago
+- Package-Sets project: https://github.com/purescript/package-sets
+- Dhall language:       https://dhall-lang.org/
```

Comment:
```suggestion
- Dhall language tour: https://docs.dhall-lang.org/tutorials/Language-Tour.html
```
In the same philosophy as above, linking to the website might be a bit distracting (albeit neat because of the inline interactive examples, etc), while I suspect the main question one might have about it is "how to I use this thing", so the Language Tour might be more useful